README.md

```diff
@@ -1,3 +1,6 @@
-# Helper app
+# Helper app 
+
+[![CI](https://github.com/fungss/Dojo-Helper-App/actions/workflows/ci.yaml/badge.svg)](https://github.com/fungss/Dojo-Helper-App/actions/workflows/ci.yaml)
```

Comment:
This gets the badge from your fork not the source code, also make sure this a relevent branch (like main) and not this branch.

Reply:
The badge won't work until the CI has ran at least once, The repo (and therefor the status) is all public so it will look the same after the CI runs.